components/layout_2020/display_list/stacking_context.rs

```diff
@@ -1451,22 +1452,24 @@ impl BoxFragment {
     }
 
     /// Returns true if the given style contains a transform that is not invertible.
-    fn has_non_invertible_transform_or_zero_scale(&self, containing_block: &Rect<Length>) -> bool {
+    fn has_non_invertible_transform_or_zero_scale(&self, containing_block: &Rect<Au>) -> bool {
         let list = &self.style.get_box().transform;
-        match list.to_transform_3d_matrix(Some(containing_block)) {
+        match list.to_transform_3d_matrix(Some(&au_rect_to_length_rect(containing_block))) {
```

Comment:
I had to a convert here because it is expected to be `ComputedType` as can be noticed [here](https://github.com/servo/stylo/blob/ac65c5a27c1b9faf9e7bb5bbcb3a4837a810ef6b/style/values/generics/transform.rs#L572). It was trickier to keep `&Rect<Length>` as it was!

same of `calculate_transform_matrix` in this same file!

Reply:
Yeah, this makes sense I think. Transforms use f32.